docs/source/conf.py

```diff
@@ -17,14 +17,15 @@
 # -- General configuration ---------------------------------------------------
```

Comment:
Wouldn't that update the date any time someone builds the documentation? If our last update was in 2028 and someone builds it in 2035, it would be incorrect to list a copyright of 2035.

Reply:
Well _technically_ the copyright text for the release would then be unique and _could_ be considered part of the copyrighted material… 😂 but that's a good point. 🙈